run.py

```diff
@@ -35,6 +35,5 @@ def create_tables():
     from app.database.sqlalchemy_extension import db
     db.create_all()
 
-
 if __name__ == "__main__":
-    application.run(port=5000)
+    application.run(port=5000, host='0.0.0.0')
```

Comment:
Backend won't work in browser without ```host='0.0.0.0'```. Tried with ```host='127.0.0.1'``` but this didn't work.

Reply:
are you using the port in address?

Reply:
Yes. The API is visible only on 0.0.0.0:5000. You can see the swagger screenshot. According to this article 
https://pythonspeed.com/articles/docker-connection-refused/ found by @bartek-pacia , this step is necessary